From: Matt Benjamin <mbenjamin@redhat.com>
Date: Wed, 14 Dec 2016 16:31:19 -0500
Subject: use std::map

(adaptation commit for #7639)

Signed-off-by: Matt Benjamin <mbenjamin@redhat.com>
(cherry picked from commit 9fd29b440990c3458e4e65f22733d9da0c2606bc)

Resolves: rhbz#1298571
---
 src/rgw/rgw_rest.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/rgw/rgw_rest.h b/src/rgw/rgw_rest.h
index ba22c1d..4bb4c28 100644
--- a/src/rgw/rgw_rest.h
+++ b/src/rgw/rgw_rest.h
@@ -6,8 +6,6 @@
 
 #define TIME_BUF_SIZE 128
 
-#include <boost/utility/string_ref.hpp>
-#include <boost/container/flat_set.hpp>
 #include "common/sstring.hh"
 #include "common/ceph_json.h"
 #include "include/assert.h" /* needed because of common/ceph_json.h */
@@ -441,7 +439,7 @@ class RGWLibIO;
 
 class RGWREST {
   using x_header = basic_sstring<char, uint16_t, 32>;
-  boost::container::flat_set<x_header> x_headers;
+  std::set<x_header> x_headers;
   RGWRESTMgr mgr;
 
   static int preprocess(struct req_state *s, RGWClientIO *sio);
